core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -205,6 +205,33 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
 
   <script src="{{TEMPLATE_DIR_PREFIX}}/expressions/expressionInterpolationService.js"></script>
 
+  <!-- these are the SchemaBased files that were moved from base index.html-->
+  <script src="{{TEMPLATE_DIR_PREFIX}}/components/forms/schema_editors/SchemaBasedBoolEditorDirective.js"></script>
```

Comment:
Oh, actually, that makes perfect sense, since that field is indeed an "editor UI". Ignore what I said. Thanks for clarifying!